src/sync/outgoingSyncUtils.js

```diff
@@ -233,3 +259,15 @@ function getTimeString(date) {
   if (!date || typeof date !== 'object') return '00:00:00';
   return date.toTimeString().substring(0, 8);
 }
+
+// Tries to get a value that is known to potentially lead to crash
+// If path on the record returns null, throw an error with prototype
+// extended with 'isSafe' set to true so sync knows to continue
+function safeGet(record, path) {
+  const value = _.get(record, path);
```

Comment:
I am not sure why we want to use underscore library.  It would be quite easy to implement a safeGet(record, ...paths).

Reply:
Axed